inc/saimacsec.h

```diff
@@ -313,6 +327,24 @@ typedef enum _sai_macsec_attr_t
      */
     SAI_MACSEC_ATTR_MAX_SECURE_ASSOCIATIONS_PER_SC,
 
+    /**
+     * @brief MACSEC POST status
+     * Attribute to query the status of POST for a MACSEC engine
+     *
+     * @type sai_macsec_post_status_t
+     * @flags READ_ONLY
+     */
+    SAI_MACSEC_ATTR_POST_STATUS,
+
+    /**
+     * @brief Setting the value to true will start the post on all the ports serviced by this MACSEC engine
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_MACSEC_ATTR_ENABLE_POST,
+
```

Comment:
MACSec and IPSec object level POST enable is mainly for diagnostics purposes.
Add comments in documentation and attribute clarifying this.